winml/adapter/winml_adapter_c_api.h

```diff
@@ -294,7 +294,11 @@ struct WinmlAdapterApi {
 	 * This api is used to add the DML EP to OrtSessionOptions.
     */
   OrtStatus*(ORT_API_CALL* OrtSessionOptionsAppendExecutionProvider_DML)(
```

Comment:
>OrtSessionOptionsAppendExecutionProvider_DML [](http://example.com/codeflow?start=27&length=44)

This API was moved to use the extension APIs in dml_provider_factory. So it looks like this is just unused now... 

Can probably just delete this...